core/tools/generate_names.cc

```diff
@@ -488,6 +489,7 @@ NameDef names[] = {
     {"NonForcingConstants", "NonForcingConstants", true},
     {"VERSION", "VERSION", true},
     {"Thread", "Thread", true},
+    {"Test", "Test", true}, // TODO will this break people?
```

Comment:
Nope. Names don't have semantic meaning, so they can be shared between things.
```suggestion
    {"Test", "Test", true},
```